kubernetes/base/statefulset.yaml

```diff
@@ -17,6 +17,9 @@ spec:
     spec:
       serviceAccountName: chat-service
       volumes:
+      - name: statsd-mapping
```

Comment:
```suggestion
      # Configuration files for the statsd_exporter sidecar
      - name: statsd-mapping
```
Although, where does this configmap come from ?

Reply:
I forgot to push the changes, pushing them now